miio/device.py

```diff
@@ -114,7 +114,8 @@ class Device(metaclass=DeviceGroupMeta):
     This class should not be initialized directly but a device-specific class inheriting
     it should be used instead of it."""
     def __init__(self, ip: str = None, token: str = None,
-                 start_id: int=0, debug: int=0, lazy_discover: bool=True) -> None:
+                 start_id: int=0, debug: int=0, lazy_discover: bool=True,
+                 device_info: DeviceInfo=None) -> None:
```

Comment:
What's the use-case for passing this as a parameter? Using the results from discovery directly or..?